model-optimizer/mo/utils/logger.py

```diff
@@ -92,7 +92,7 @@ def wrapper(*args, **kwargs):
             assert arg in kwargs, msg.format(arg, 'is missing')
             assert kwargs[arg] is not None, msg.format(arg, 'should not be None')
 
-        if kwargs['graph'].graph['cmd_params'].progress:
+        if 'progress' in kwargs['graph'].graph['cmd_params']:
```

Comment:
Is `progress` always `True`? These conditions are not equivalent, maybe, it should be
`if 'progress' in kwargs['graph'].graph['cmd_params'] and kwargs['graph'].graph['cmd_params'].progress:`

Reply:
Done, condition updated.